### ISSUE TITLE
Package Control not properly installing package on SublimeText2

### ISSUE THREAD

**Issue by DirtGrubDylan** [KristoforMaynard/SublimeAutoDocstring#3]
So going through the motions with Package Control on ST2:
adding url repo
installing package

I have found that package control is not properly installing/detecting SublimeAutoDocstring folder. When I open default settings under `Preferences > Package Settings > AutoDocstring > Settings - Default`, it is blank.

And when I hit the commands (ctrl + alt + ') to add docstring, nothing happens.

Is there a fix for this?

**Comment by KristoforMaynard** [comment]
This plugin was really written for ST3, and given how much better it is than ST2, I'm poorly motivated to maintain backward compatibility. That said, I believe this plugin could be made compatible with ST2. Changes that immediately come to mind are the file locations and python 2.6's lack of an OrderedDict (there is a backport floating around the internet for this). I also used python's ast module for parsing function parameters, and I'm not sure if that code works "as is" with python 2.6.

Long story short, ST2 compatibility should be doable, but I'll wait for a pull request before I give it any more thought.
